script.module.pyserial/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyserial"
+       name="pyserial"
+       version="3.4.0"
+       provider-name="feffe">
+  <requires>
+    <import addon="xbmc.python" version="2.25.0"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Python serial port access library</summary>
+    <description lang="en_GB">Packed for KODI from https://github.com/pyserial/pyserial</description>
+    <platform>all</platform>
+    <language></language>
+    <license>BSD-3-Clause</license>
+    <forum></forum>
+    <website>https://github.com/pyserial/pyserial</website>
+    <source>https://github.com/fredrik-eriksson/script.module.pyserial</source>
```

Comment:
I think the original module source repository fits better here but I'm fine as it is.

Reply:
Right, LICENSE and provider-name fixed, I'd like to keep the source as is as it contains the actual source code that is used to produce the plugin. If I mess up with the packaging and the source happens to differ from the original, I think the blame should go to me and not the original author :)

Reply:
I'm okay with that